napari/layers/base/base.py

```diff
@@ -230,6 +230,11 @@ def __init__(
         if name is None and data is not None:
             name = magic_name(data)
 
+        if scale and 0 in scale:
+            raise Exception(
```

Comment:
should be a `ValueError` (try to use as specific of an Exception subclass as is appropriate)